mm2src/lp_ordermatch.rs

```diff
@@ -2999,6 +3064,19 @@ struct SetPriceReq {
     rel_nota: Option<bool>,
 }
 
+#[derive(Deserialize)]
+struct MakerOrderUpdateReq {
+    uuid: Uuid,
+    new_price: Option<MmNumber>,
```

Comment:
Just out of curiosity, new_price is not supposed to be mandatory in this rpc call ? What happens if i try to use `update_maker_order` without specifying a new price ?

Reply:
It leaves the old price as is. But validates it against the volume, etc..
https://github.com/KomodoPlatform/atomicDEX-API/blob/6120932eeae0ce7ccc0eb6e0b5c146802d9efdb3/mm2src/lp_ordermatch.rs#L3417